From 661d603b65385f62f372acd2017e5af2e0f0cd50 Mon Sep 17 00:00:00 2001
From: Alex Richardson <Alexander.Richardson@cl.cam.ac.uk>
Date: Thu, 2 Nov 2017 14:24:23 +0000
Subject: [PATCH] build: use #ifdef for ENABLE_NLS check.

config.h will have either define ENABLE_NLS or not define it. If it is
not defined we get a -Wundef warning due to using #if with an
undefined macro
---
 src/gettext.h | 2 +-
 src/main.c    | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/gettext.h b/src/gettext.h
index ea67f308..3f629611 100644
--- a/src/gettext.h
+++ b/src/gettext.h
@@ -20,7 +20,7 @@
 #define _LIBGETTEXT_H 1
 
 /* NLS can be disabled through the configure --disable-nls option.  */
-#if ENABLE_NLS
+#ifdef ENABLE_NLS
 
 /* Get declarations of GNU message catalog functions.  */
 # include <libintl.h>
diff --git a/src/main.c b/src/main.c
index bf8c44d8..65285a02 100644
--- a/src/main.c
+++ b/src/main.c
@@ -197,7 +197,7 @@ int flex_main (int argc, char *argv[])
 /* Wrapper around flex_main, so flex_main can be built as a library. */
 int main (int argc, char *argv[])
 {
-#if ENABLE_NLS
+#ifdef ENABLE_NLS
 #if HAVE_LOCALE_H
 	setlocale (LC_MESSAGES, "");
         setlocale (LC_CTYPE, "");
